.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,14 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Frequently Asked Questions
+    url: https://github.com/OpenRA/OpenRA/wiki/FAQ#common-issues
```

Comment:
Shouldn't this link to https://github.com/OpenRA/OpenRA/wiki/FAQ#frequently-asked-questions (the table of content)? We already refer to common issues in the header of the crash template.